Project.swift

```diff
@@ -0,0 +1,49 @@
+import ProjectDescription
+
+let deploymentTarget: DeploymentTarget = .macOS(targetVersion: "10.15")
+let baseSettings: SettingsDictionary = ["EXCLUDED_ARCHS": "arm64"]
```

Comment:
why exactly is this necessary?

Reply:
Because otherwise, it fails to build one of the dependencies for some reason.

Reply:
Uhm this means folks with Apple Silicon will run it in Rosetta. Might be worth understanding why this isn't working / building.